nexus/tests/integration_tests/console_api.rs

```diff
@@ -334,23 +334,32 @@ async fn test_session_me(cptestctx: &ControlPlaneTestContext) {
         .execute()
         .await
         .expect("failed to get current user")
-        .parsed_body::<views::SessionUser>()
+        .parsed_body::<views::User>()
         .unwrap();
 
-    assert_eq!(priv_user, views::SessionUser { id: USER_TEST_PRIVILEGED.id() });
+    assert_eq!(
+        priv_user,
+        views::User {
+            id: USER_TEST_PRIVILEGED.id(),
+            display_name: USER_TEST_PRIVILEGED.external_id.clone()
+        }
+    );
 
     // make sure it returns different things for different users
     let unpriv_user = NexusRequest::object_get(testctx, "/session/me")
         .authn_as(AuthnMode::UnprivilegedUser)
         .execute()
         .await
         .expect("failed to get current user")
-        .parsed_body::<views::SessionUser>()
+        .parsed_body::<views::User>()
         .unwrap();
 
     assert_eq!(
         unpriv_user,
-        views::SessionUser { id: USER_TEST_UNPRIVILEGED.id() }
+        views::User {
+            id: USER_TEST_UNPRIVILEGED.id(),
+            display_name: USER_TEST_PRIVILEGED.external_id.clone()
+        }
```

Comment:
Tried do so some cute `views::User::from(USER_TEST_UNPRIVILEGED)` but there was no combination of `&` and `*` and whatever else that could make it work.